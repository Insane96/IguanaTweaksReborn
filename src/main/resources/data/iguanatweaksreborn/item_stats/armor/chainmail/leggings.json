{
	"item": "minecraft:chainmail_leggings",
	"durability": 60,
	"armor": 3,
	"toughness": 1
}