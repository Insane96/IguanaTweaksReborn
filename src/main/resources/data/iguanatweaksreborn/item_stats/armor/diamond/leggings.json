{
	"item": "minecraft:diamond_leggings",
	"durability": 225,
	"armor": 5,
	"toughness": 1,
	"knockback_resistance": 0.03
}