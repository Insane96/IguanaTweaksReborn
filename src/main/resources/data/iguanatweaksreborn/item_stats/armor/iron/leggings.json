{
	"item": "minecraft:iron_leggings",
	"durability": 90,
	"armor": 4
}