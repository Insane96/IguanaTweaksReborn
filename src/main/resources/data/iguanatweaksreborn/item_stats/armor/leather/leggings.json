{
	"item": "minecraft:leather_leggings",
	"durability": 90,
	"armor": 2,
	"toughness": 1
}