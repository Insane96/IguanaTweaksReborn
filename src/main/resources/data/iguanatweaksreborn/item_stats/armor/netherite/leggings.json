{
	"item": "minecraft:netherite_leggings",
	"durability": 210,
	"armor": 6,
	"toughness": 2,
	"knockback_resistance": 0.05
}