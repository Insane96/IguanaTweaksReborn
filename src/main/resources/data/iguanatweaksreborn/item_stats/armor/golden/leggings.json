{
	"item": "minecraft:golden_leggings",
	"durability": 90,
	"armor": 3,
	"toughness": 1
}